cdap-ui/package.json

```diff
@@ -203,6 +204,7 @@
     "q": "1.5.1",
     "query-string": "6.9.0",
     "react": "16.12.0",
+    "prop-types": "15.5.10",
```

Comment:
nit: Should this be 15.7.2? We upgraded this in develop but do we have to downgrade this?